src/account/sync/mod.rs

```diff
@@ -389,55 +389,53 @@ impl SyncedAccount {
             self.select_inputs(*transfer_obj.amount(), &account, transfer_obj.address())?;
 
         let mut utxos = vec![];
-        let mut output_paths = vec![];
+        let mut address_index_recorders = vec![];
+
+        let account_path =
+            BIP32Path::from_str(&format!("m/44H/4218H/{}H", account.index())).unwrap();
+
         for input_address in &input_addresses {
-            let address = input_address.address();
+            let address_outputs = input_address.outputs();
+            let mut outputs = vec![];
             let address_path = BIP32Path::from_str(&format!(
```

Comment:
A little thing here. It might be better to use a constant to store `m/44H/4218H/{}`, but we can modify all this kind of things together in the future refactory phase.

Reply:
I tried it but it wasn't working, not sure why.